src/tag.rs

```diff
@@ -1425,4 +1428,12 @@ mod tests {
         tag_file.seek(io::SeekFrom::Start(0)).unwrap();
         assert!(!Tag::remove_from(&mut tag_file).unwrap());
     }
+
+    #[test]
+    fn test_junk_data() {
```

Comment:
For this test, could you please:

* Rename this test to test_issue_39?
* Add a link to #39?
* Copy quiet.mp3 to a temp file and write to that? Tip: `tempfile::NamedTempFile::new()`
* Spin up ffprobe and check its output for whether the error is absent?